site/docs/transform/aggregate.md

```diff
@@ -125,6 +125,8 @@ The supported **aggregation operations** are:
 
 ## Argmin / Argmax
 
+Sometimes, you may not want to find the minimum or maximum of a field but the value from a field corresponds to the minimum or maximum value in another fields. In these cases you can use the argmin and argmax aggregates.
```

Comment:
"Sometimes, you may not want to find the minimum or maximum of a field, but instead the value from a field that corresponds to the minimum or maximum value in another field." ?

Reply:
Thanks